client/my-sites/site-settings/form-analytics-stores.jsx

```diff
@@ -77,6 +77,13 @@ class FormAnalyticsStores extends Component {
 			{
 				key: 'ec_track_add_to_cart',
 				label: translate( 'Add to cart events' ),
+				explanation: translate(
+					'Before enabling these, turn on eCommerce in your Google Analytics dashboard.'
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Before enabling, turn on enhanced eCommerce in your Google Analytics dashboard.' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).